sklearn/compose/tests/test_column_transformer.py

```diff
@@ -525,7 +526,11 @@ def test_make_column_transformer():
     # XXX remove in v0.22
     with pytest.warns(DeprecationWarning,
                       match='`make_column_transformer` now expects'):
-        make_column_transformer(('first', scaler))
+        ct1 = make_column_transformer(([0], norm))
+    ct2 = make_column_transformer((norm, [0]))
+    X_array = np.array([[0, 1, 2], [2, 4, 6]]).T
```

Comment:
I think that @jorisvandenbossche meant the correct order of (transformer, column), not the correct order of data... but I suppose this confirms that it's behaving as expected at least.

Reply:
and actually I'm addressing @ogrisel's comment here, see https://github.com/scikit-learn/scikit-learn/pull/12626#discussion_r235175726 :)

Reply:
Yes, that tests that as well :)